### ISSUE TITLE
Update schema URL in npm package

### ISSUE THREAD

**Issue by mode777** [robertlong/gltf-typescript-generator#2]
In the repository, the default schema URL has been updated, but the changes do not seem to be present in the npm package (0.0.3), as I get...

```
Error: Error downloading https://rawgit.com/KhronosGroup/glTF/2.0/specification/2.0/schema/glTF.schema.json
```

I guess it needs to be updated.

**Comment by TimvanScherpenzeel** [comment]
It should be changed to: `https://raw.githubusercontent.com/KhronosGroup/glTF/master/specification/2.0/schema/glTF.schema.json`

**Comment by aaron-human** [comment]
Judging by the source, the current work-around is to specify the above URL manually as the second CLI parameter:
```
gltf-typescript-generator <outputPath> https://raw.githubusercontent.com/KhronosGroup/glTF/master/specification/2.0/schema/glTF.schema.json
```

**Comment by robertlong** [comment]
Fixed in v0.0.4
